doc/whats-new.rst

```diff
@@ -51,6 +51,7 @@ Internal Changes
   (:pull:`7847`) By `Tom Nicholas <https://github.com/TomNicholas>`_.
 - Xarray now uploads nightly wheels to https://pypi.anaconda.org/scientific-python-nightly-wheels/simple/.
   By `Martin Fleischmann <https://github.com/martinfleis>`_.
+- Added an exception catch for ``AttributeError`` along with ``ImportError`` when duck typing the dynamic imports in pycompat.py. This catches some name collisions between packages.
```

Comment:
```suggestion
- Added an exception catch for ``AttributeError`` along with ``ImportError`` when duck typing the dynamic imports in pycompat.py. This catches some name collisions between packages. (:issue:`7870`, :pull:`7874`)
```
It's good practice to reference the issue the change was addressing, just in case anyone needs to look this up later.